Frontend/ui-customer/src/MerchantConfigs/DefaultConfig.purs

```diff
@@ -258,7 +259,9 @@ config =
               { goToNearestPointWithinRadius : 8.0
               , showHotSpotsWithinRadius : 150.0
               , enableHotSpot : JB.jBridgeMethodExists "locateOnMapV2"
-              }
+              },
+            editPickUpThreshold : 100.0,
+            editPickUpRadius : 100.0 --in meters
```

Comment:
can we use one of the two variables at all places instead of creating two different variables.